templates/data/docs/opensearch/iaas/document.html

```diff
@@ -0,0 +1,11 @@
+{% with
```

Comment:
Double heading on this one 
![Screenshot from 2023-07-03 10-40-46](https://github.com/canonical/ubuntu.com/assets/17607612/753ae1e2-2a80-4d4a-9f41-0d5a5099ba10)